lib/ui/platform_dispatcher.dart

```diff
@@ -1300,10 +1305,12 @@ class ViewConfiguration {
     this.padding = WindowPadding.zero,
     this.gestureSettings = const GestureSettings(),
     this.displayFeatures = const <DisplayFeature>[],
-  });
+  }) : assert(!(window != null && view != null)),
```

Comment:
could be simplified to `assert(window == null || view == null)`, no?

Reply:
vacation...

Reply:
23be98b9c5f2eee6ceb810a7f6f9a67ea3383555 should fix it. Thanks!